src/tab.group.ts

```diff
@@ -152,9 +150,6 @@ export default class TabGroup extends LitElement {
 
   #localize = new LocalizeController(this);
 
-  // Theshold (in px) used to determine when to display overflow buttons.
-  #overflowButtonsScrollDelta = 1;
```

Comment:
This seems to be a hack to work around using `scrollWidth` and `getBoundingClientRect().width` together. The former is an integer, the latter a float. And `getBoundingClientRect().width` was apparently used to work around `clientWidth` a different number than 

TODO